test/Tooltip.test.js

```diff
@@ -254,6 +281,36 @@ describe("Tooltip", () => {
 			proclaim.isTrue(buttonEl.hasAttribute('title'));
 			proclaim.isTrue(buttonEl.hasAttribute('role'));
 		});
+
+// 		// <div id="demo-tooltip-insertion-test-1">
+// // 			<div class='tooltip-target' id="demo-tooltip-insertion-test-1-target">
+// // 				Thing to point the tooltip at.
+// // 			</div>
+// // 		</div>
+
+// // 		<div id="demo-tooltip-insertion-test-2">
+// // 			<div class='tooltip-target' id="demo-tooltip-insertion-test-2-target">
+// // 				Thing to point the tooltip at.
+// // 			</div>
+// // 			<div></div>
+// // 		</div>
+
+
+
+
+// 		it("Inserts adjacent to target element when target has no next sibling", () => {
```

Comment:
was so long ago I looked at this, but as I recall, getting this test to work was tricky, which is why I originally asked for help. But these seem like important tests to have